components/contextualhelp/index.css

```
@@ -11,42 +11,88 @@
  * governing permissions and limitations under the License.
  */
 
+.spectrum-ContextualHelp {
+	--spectrum-contextual-help-title-sans-serif-font-family: var(--spectrum-sans-font-family-stack);
+	--spectrum-contextual-help-title-font-weight: var(--spectrum-title-sans-serif-font-weight);
+	--spectrum-contextual-help-title-font-style: var(--spectrum-title-sans-serif-font-style);
+	--spectrum-contextual-help-title-line-height: var(--spectrum-title-line-height);
+
+	--spectrum-contextual-help-title-color: var(--spectrum-title-color);
+
+	--spectrum-contextual-help-body-sans-serif-font-family: var(--spectrum-sans-font-family-stack);
+	--spectrum-contextual-help-body-sans-serif-font-weight: var(--spectrum-body-sans-serif-font-weight);
+	--spectrum-contextual-help-body-sans-serif-font-style: var(--spectrum-body-sans-serif-font-style);
+	--spectrum-contextual-help-body-line-height: var(--spectrum-line-height-200);
+
+	--spectrum-contextual-help-body-color: var(--spectrum-body-color);
+
+	--spectrum-contextual-help-min-inline-size: var(--spectrum-contextual-help-minimum-width);
+	--spectrum-contextual-help-link-spacing: var(--spectrum-spacing-300);
+	--spectrum-contextual-help-padding: var(--spectrum-spacing-400);
+
+	--spectrum-contextual-help-border-radius: var(--mod-contextual-help-border-radius, var(--mod-popover-corner-radius));
+
+	/* @passthrough start */
+	/* stylelint-disable-next-line spectrum-tools/no-unused-custom-properties -- used to assign Popover mods */
+	--spectrum-picker-popover-animation-distance: var(--spectrum-component-to-menu-medium);
+
+	--mod-popover-animation-distance: var(--spectrum-picker-popover-animation-distance);
```

```suggestion
	--mod-popover-animation-distance: var(--spectrum-spacing-100);
```

Looking at the token specs, I think we can simply set the spacing token as suggested. I don't think you need to set it in the `spectrum-ContextualHelp-popover` selector below either - works without it.